vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -63,7 +63,8 @@ void Publisher::RefreshPublisher(
         // attempt to process any pending contributions for
         // unverified publishers.
         if (status == type::PublisherStatus::UPHOLD_VERIFIED ||
-            status == type::PublisherStatus::BITFLYER_VERIFIED) {
+            status == type::PublisherStatus::BITFLYER_VERIFIED ||
```

Comment:
Would be safer as a switch statement

Reply:
fixed: https://github.com/brave/brave-core/pull/8566/files#diff-ea6a5f02c609dc2fcd0528606219ad12ba79823e8c0d5b095c67f9f64bb157fa